### ISSUE TITLE
Update lodash library to latest version

### ISSUE THREAD

**Issue by Biktop** [balderdashy/sails#3078]
Could you update lodash library to latest (v3.10)?
While I can use own version in controllers and models then it's difficult to use own version in view (ejs).

**Comment by tjwebb** [comment]
If you'd like to submit a PR with this improvement, we'd gladly merge it. Thanks.

**Comment by kanecko** [comment]
+1
This would also be nice for client/server consistency.

**Comment by Biktop** [comment]
I've discovered differences in function _.clone between v2.4.1 and v3.10.1 in case if we passed function as value. So in v2.4.1 _.clone just returns the passed value whereas v3.10.1 returns {}. Unfortunately I've found a lot of places in sails where _.clone is used to clone functions. Now I'm not sure that I understand why sails tries clone functions if _.clone just return the same value.

**Comment by sailsbot** [comment]
Thanks for posting, @Biktop.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
